lib/src/network/explorer/explorer_api.dart

```diff
@@ -1078,168 +1179,105 @@ class InputUtxo {
-  final String txnHash;
-  final int txnTime;
-  final String type;
   final int weight;
+  final int priority;
 
   factory ValueTransferInfo.fromJson(Map<String, dynamic> data) {
-    List<InputUtxo> inputs = [];
-    List<ValueTransferOutput> outputs = [];
-    List<dynamic> inputAddresses = data['input_addresses'] ?? [];
-    List<dynamic> outputAddresses = data['output_addresses'] ?? [];
-    Map<String, dynamic>? inputUxtos = data['input_utxos'];
-    inputAddresses.forEach((element) {
-      if (inputUxtos != null && inputUxtos.containsKey(element[0])) {
-        var _sub = inputUxtos[element[0]]!.first;
-
-        String outputPointer = '${_sub[1]}:${_sub[2]}';
-
-        inputs.add(
-          InputUtxo(
-            address: element[0] as String,
-            input:
-                Input(outputPointer: OutputPointer.fromString(outputPointer)),
-            value: element[1] as int,
-          ),
-        );
-      }
-    });
-
-    outputAddresses.forEach((element) {
-      Address address = Address.fromAddress(element[0]);
-
-      outputs.add(ValueTransferOutput(
-          pkh: address.publicKeyHash!,
-          timeLock: element[2],
-          value: element[1]));
-    });
     return ValueTransferInfo(
-      blockHash: data["block_hash"],
-      fee: data["fee"] ?? 0,
-      priority: data["priority"] ?? 0,
-      status: data["status"],
-      txnEpoch: data["txn_epoch"],
-      txnHash: data["txn_hash"] ?? '',
-      txnTime: data["txn_time"] ?? 0,
-      type: data["type"],
-      weight: data["weight"] ?? 0,
-      inputs: inputs,
-      outputs: outputs,
-    );
+        epoch: data["epoch"],
+        timestamp: data["timestamp"],
+        hash: data["hash"],
+        inputAddresses: data["input_addresses"],
+        outputAddresses: data["output_addresses"],
+        value: data["value"],
+        fee: data["fee"],
+        weight: data["weight"],
+        priority: data["priority"]);
   }
 
   factory ValueTransferInfo.fromDbJson(Map<String, dynamic> data) {
-    List<InputUtxo> inputs = [];
-    List<ValueTransferOutput> outputs = [];
-    List<dynamic> out = data['outputs'];
-    List<dynamic> inp = data['inputs'];
-    inp.forEach((element) {
-      inputs.add(
-        InputUtxo(
-          address: element['pkh'],
-          input: Input(
-              outputPointer:
-                  OutputPointer.fromString(element['output_pointer'])),
-          value: element['value'],
-        ),
-      );
-    });
-    out.forEach((element) {
-      outputs.add(ValueTransferOutput(
-        pkh: PublicKeyHash.fromAddress(element['pkh']),
-        timeLock: element['time_lock'],
-        value: element['value'],
-      ));
-    });
-
     return ValueTransferInfo(
-      blockHash: data["block_hash"],
+      epoch: data["epoch"],
+      timestamp: data["timestamp"],
+      hash: data["hash"],
+      inputAddresses: data["input_addresses"],
+      outputAddresses: data["output_addresses"],
+      value: data["value"],
```

Comment:
Why are we updating the structure in `ValueTransferInfo` and not in `MintEnty`?
The `InputUtxo` list and `ValueTransferOutput` list is not accesible anymore and there is a new field added called `value`, which is also missing in `MintEntry`.

Reply:
Because `ValueTransferInfo` and `MintInfo` have different schema. In the case of  `MintInfo`, I just had to add a field, but I could keep the same interface we had before. I'm going to double-check to ensure I'm not missing any field you may need in `ValueTransferInfo`.

Reply:
I understand that the call to get the the vtt info needs to be done anyway `/api/search/hash`. `/api/address/value-transfers` now include more fields, before it just return the transaction hashes. The schema of the `ValueTransfer Info` is not correct. It definitely updated, but the schema does not match the updated one.
```value_transfer: {
epoch*: integer
timestamp*: integer
hash*: string
block*: string
confirmed*: boolean
reverted*: boolean
input_addresses*: [string]
input_utxos*: [{
address*: string
value*: integer
Constraints: Min 0
input_utxo*: string
}]
inputs_merged*: [{
address*: string
value*: integer}]
output_addresses*: [string]
output_values*: [integer]
Constraints: Min 1
timelocks*: [integer]
utxos*: [{
address*: string
value*: integer
timelock*: integer
locked*: boolean
}]
utxos_merged*: [{
address*: string
value*: integer
timelock*: integer
locked*: boolean
}]
fee*: integer
value*: integer
priority*: integer
weight*: integer
true_output_addresses*: [string]
change_output_addresses*: [string]
true_value*: integer
change_value*: integer
}```